problem_builder/mentoring.py

```diff
@@ -847,13 +857,19 @@ class MentoringWithExplicitStepsBlock(BaseMentoringBlock, StudioContainerWithNes
         enforce_type=True
     )
 
-    editable_fields = ('display_name',)
+    editable_fields = ('display_name', 'max_attempts', 'extended_feedback')
 
     @lazy
     def questions(self):
         """ Get the usage_ids of all of this XBlock's children that are "Questions" """
         return list(chain.from_iterable(self.runtime.get_block(step_id).steps for step_id in self.steps))
 
+    def get_questions(self):
```

Comment:
@itsjeyd It is not obvious how `questions` differs from `get_questions` only by looking at the method names. Maybe rename `questions` to `question_ids` and `get_questions` to `questions`?